flowcraft/generator/templates/bcalm.nf

```diff
@@ -0,0 +1,29 @@
+// Check parameter
+if ( !params.bcalmKmerSize{{ param_id }}.toString().isNumber() ){
+    exit 1, "'bcalmKmerSize{{ param_id }}' parameter must be a number. Provided value: '${params.bcalmKmes%rSize{{ param_id }}}'"
+}
+
+// Clear
+clear = params.clearInput{{ param_id }} ? "true" : "false"
```

Comment:
Depending on whether you want to add the code to remove temporary files or not, the option were can stay or be removed as well.